docs/sphinx/manual/building_hpx.rst

```diff
@@ -1269,3 +1271,25 @@ How to install |hpx| on Fedora distributions
 .. include:: ../../generated/cmake_toolchains.rst
 
 .. include:: ../../generated/cmake_variables.rst
```

Comment:
Could you move this section just above the two includes and directly after the fedora section?

Reply:
Sounds good, even I thought the placement was a bit off. I'll change it accordingly.